src/dotnet/CarbonAware.WebApi.Tests/unitTests/TestHelpers.cs

```diff
@@ -0,0 +1,26 @@
+namespace CarbonAware.WepApi.UnitTests;
+
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+
+/// <summary>
+/// Tests helpers for all WebAPI specific tests.
+/// </summary>
+public static class TestHelpers
+{
+    public static void AssertStatusCode(IActionResult result, int code)
+    {
+        if (result is not ObjectResult obj)
+        {
+            // Certain result, like NoContent(), will only return a StatusCodeResult
```

Comment:
This comment is probably better served as XML doc content "remarks" for this function